### ISSUE TITLE
Space and and Sizes bug

### ISSUE THREAD

**Issue by arnaudjnn** [chakra-ui/chakra-ui#1669]
Since I updated to 1.0.0-rc.2 all the space, sizes and fontSizes are not working anymore.

For example:
```JSX
<Grid gridGap={5} />
```
render 5px instead 1.25rem as defined in the default theme.

Why?

**Comment by ljosberinn** [comment]
hi, could you please add a reproduction? not seeing this here: https://codesandbox.io/s/cocky-sun-1tgku?file=/src/index.js

**Comment by arnaudjnn** [comment]
Hi @ljosberinn 

Here is a reproduction: https://codesandbox.io/s/boring-river-wtk2z?file=/pages/index.tsx

Thanks

**Comment by ljosberinn** [comment]
thank you, we'll look into this

**Comment by ljosberinn** [comment]
is this a reproduction of something you see locally too? it looks like codesandbox ignores your `_app` entirely, as you can see here:

![image](https://user-images.githubusercontent.com/29307652/90243072-80dc3b00-de2e-11ea-9125-5ff248098ac5.png)

ChakraProvider is missing, and thus CSSReset, Theme, etc.

**Comment by arnaudjnn** [comment]
Yes the issue is locally too. Actually I have another project with 1.0.0-rc.0. It tried to copy/past the @chakra folder from node_modules and it works. But it doesn't work when I delete yarn.lock and node_module and run yarn again (for both 1.0.0-rc.0 and 1.0.0-rc.2)

**Comment by arnaudjnn** [comment]
<img width="1280" alt="Screenshot 2020-08-14 at 13 16 02" src="https://user-images.githubusercontent.com/25635480/90244081-57bcaa00-de30-11ea-9119-3b11677e3ff3.png">

Even when you import a basic component like ```Switch``` it doesn't display normally

**Comment by ljosberinn** [comment]
> @chakra-ui/core > @chakra-ui/tabs@1.0.0-rc.2" has incorrect peer dependency "@chakra-ui/system@1.0.0-rc.1".

that gladly doesnt matter and was just a mistake before publishing rc2, its fixed already :)

**Comment by jmiazga** [comment]
I tried the [codesandbox above](https://codesandbox.io/s/boring-river-wtk2z?file=/pages/index.tsx), and I see that grid-gap=1.25rem

<img width="249" alt="image" src="https://user-images.githubusercontent.com/181441/90248149-1f07db00-de06-11ea-913b-62211799b527.png">

**Comment by ljosberinn** [comment]
I now do too, maybe a server restart or something was required on csbs side:
![image](https://user-images.githubusercontent.com/29307652/90249131-39f44280-de3a-11ea-8487-2c86d694a1b8.png)

**Comment by arnaudjnn** [comment]
I still have the issue locally...

**Comment by jmiazga** [comment]
Is your code available somewhere on github so I can check it out?

**Comment by arnaudjnn** [comment]
Here is the branch: https://github.com/Angulaire/mr-menager/tree/update-chakra

The master branch works well with the previous version

Thanks a lot.

**Comment by ljosberinn** [comment]
you're e.g. referring to this component? 😅 https://github.com/Angulaire/mr-menager/blob/update-chakra/components/common/CategoriesGrid.tsx#L11 just making sure I look at the right thing to replicate.

**Comment by arnaudjnn** [comment]
Yes I think all the space, size and fontSize use px instead the corresponding rem defined in the theme.

For example here: https://github.com/Angulaire/mr-menager/blob/update-chakra/components/common/ArticleCard.tsx

The maxHeight={48} should be 12rem instead of 48px

**Comment by jmiazga** [comment]
Where is the theme from from here? https://github.com/Angulaire/mr-menager/blob/update-chakra/pages/_app.tsx#L3

**Comment by ljosberinn** [comment]
over here: https://github.com/Angulaire/mr-menager/blob/update-chakra/theme.js#L39
there might be a spreading mistake but I compared with the default theme and I think that's ok. still should check if its intentional to use `.space` in `.sizes`

**Comment by arnaudjnn** [comment]
Yes, I customized the theme at the project root.

It's intentional, I followed the doc: https://next.chakra-ui.com/docs/theming/theme#sizes

**Comment by with-heart** [comment]
I'm not sure what's going on for you locally. If I clone the project from GitHub and run it locally, I'm seeing the correct sizes. Tried with both `1.0.0-rc.0` and `1.0.0-rc.2` with hard cache reload.

**Comment by arnaudjnn** [comment]
Here are the steps to follow:
```git clone https://github.com/Angulaire/mr-menager```
```yarn && yarn dev``` => Works well
```rm -rf node_modules && rm yarn.lock```
```yarn && yarn dev``` => There is the issue

**Comment by nonissue** [comment]
I had issues with Chakra `1.0.0-rc.2` and nextjs, but ultimately realized the problem was caused by `emotion` being installed (I cloned the nextjs `with-chakra-ui` example, which is a little out of date).

Also, in `_document.tsx` you have the following [line](https://github.com/Angulaire/mr-menager/blob/01d9a24e92661b902b070e9c1292a8c126b75c07/pages/_document.tsx#L2):

`import { jsx } from '@chakra-ui/core';`

Should that be `sx`? 

Otherwise, just wanted to chime in to say that I'm running chakra@next with next.js 9.5.2 and added your theme to my repo and though I didn't try using the grid-gap or any spacing related utilities, I was able to change the fontSizes on my project with the theme file.

**Comment by with-heart** [comment]
@nonissue We've run into that issue a few times lately. Multiple versions of `emotion` break the `ThemeProvider` functionality, causing the theme to be ignored. 

@arnaudjnn Can you try following @nonissue's recommendation and let us know if that fixes your issue?

**Comment by kfalkiewicz** [comment]
I have fresh copy of chakra-ui@next with extended theme. I added custom size to theme but it is ignored by Button styles also.
